tests/metrics/workload_time/docker_workload_time_stress.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  This test measures the complete workload of a container using docker.
+#  From calling docker until the workload is completed and the container
+#  is shutdown.
+
+# set -e
+
+[ $# -ne 3 ] && ( echo >&2 "Usage: $0 <cmd to run> <image> <runtime>"; exit 1 )
```

Comment:
Ah - but you'll need to source `test-common.bash` at the top of the script before this line to be able to use `die`.